SwiftAppTemplate/SwiftAppTemplate/ReaderListViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ReaderListViewController.swift
+//  SwiftAppTemplate
+//
+//  Created by Catriona Scott on 4/30/19.
+//  Copyright © 2019 Catriona Scott. All rights reserved.
+//
+
+import UIKit
+
+class ReaderListViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```

Comment:
hmm – just a thought, maybe we could rename this to a more generic "ObjectListViewController"? (and rename things like the `readers` property to `objects`?)